### ISSUE TITLE
Restaurant code review

### ISSUE THREAD

**Issue by mandyWW** [jefferyforbes/Multiverse-Bootcamp#1]
Overall a really slick looking website, well done!

**Accessibility:**
- Colour contrast is insufficient (see SiteImprove report)
- The font although it looks good might be hard to read for some people so I would stick to fonts which are recommended for people with dyslexia: https://www.bdadyslexia.org.uk/advice/employers/creating-a-dyslexia-friendly-workplace/dyslexia-friendly-style-guide
- Placeholder text seems to get read aloud in screen reader, might be better to just have 'url' as placeholder wording

**Code:**
- great to see the start of the ratings functionality, Rating and Review classes show me you understood Sequelize
- good documentation in server.js showing you understand how to create routes
